researchapp/application.py

```diff
@@ -1,104 +1,28 @@
-""" Holds the create_app() Flask application factory.
-More information in create_app() docstirng.
-"""
-from importlib import import_module
-import json
+''' The application.
+'''
 import os
 
 from flask import Flask
-import yaml
 
-import researchapp as app_root
-from researchapp.blueprints import all_blueprints
-from researchapp.extensions import db, configure, injector
-from researchapp.services import authorize, providers, resources
 
-APP_ROOT_FOLDER = os.path.abspath(os.path.dirname(app_root.__file__))
-TEMPLATE_FOLDER = os.path.join(APP_ROOT_FOLDER, 'templates')
-STATIC_FOLDER = os.path.join(APP_ROOT_FOLDER, 'static')
+# Create and configured application
+app = Flask(__name__)
+app.config['SYNCHRONIZER_HOST'] = os.getenv('SYNCHRONIZER_HOST')
+app.secret_key = os.getenv('FLASK_SECRET_KEY')
```

Comment:
This is for protecting cookies that get stored client-side? I think this is fine for now, but I've added https://github.com/sync-for-science/tracking/issues/79 to decide what to do with this longer-term.